src/components/graphCard/graphCardContext.js

```diff
@@ -20,104 +21,111 @@ const GraphCardContext = React.createContext(DEFAULT_CONTEXT);
 const useGraphCardContext = () => useContext(GraphCardContext);
 
 /**
- * Use Redux RHSM Actions, getGraphTally.
- *
- * @param {object} options
- * @param {Function} options.useDispatch
- * @param {Function} options.useProductGraphTallyQuery
- * @returns {Function}
- */
-const useGetGraphTally = ({
-  useDispatch: useAliasDispatch = storeHooks.reactRedux.useDispatch,
-  useProductGraphTallyQuery: useAliasProductGraphTallyQuery = useProductGraphTallyQuery
-} = {}) => {
-  const query = useAliasProductGraphTallyQuery();
-  const dispatch = useAliasDispatch();
-
-  return metrics => reduxActions.rhsm.getGraphTally(metrics, query)(dispatch);
-};
-
-/**
- * Get multiple metrics from store.
+ * Transform multiple metrics from store for chart/graph consumption.
  *
  * @param {object} options
  * @param {Function} options.useGraphCardContext
- * @param {Function} options.useSelectors
- * @param {Function} options.useProduct
- * @returns {{data: object, pending: boolean, fulfilled: boolean, dataSets: Array, error: boolean}}
+ * @param {Function} options.useSelectorsResponse
+ * @returns {{data: {}, pending: boolean, fulfilled: boolean, responses: {errorList: *[], errorId: {},
+ *     id: {}, list: *[]}, cancelled: boolean, dataSets: unknown[], message: null, error: boolean}}
  */
 const useMetricsSelector = ({
   useGraphCardContext: useAliasGraphCardContext = useGraphCardContext,
-  useSelectors: useAliasSelectors = storeHooks.reactRedux.useSelectors,
-  useProduct: useAliasProduct = useProduct
+  // useProduct: useAliasProduct = useProduct,
+  useSelectorsResponse: useAliasSelectorsResponse = storeHooks.reactRedux.useSelectorsAllSettledResponse
```

Comment:
required:
clean up commented code

note:
the use of "all settled" is going to alter general load behavior, especially around products like RHOSAK where we have multiple standalone graphs. we'll keep an eye on it once we flip RHEL over

Reply:
note:
yes and no, running the tests, yes this does later behavior and no, as predicated earlier it's based on each "graph card". 

So in the event of a graph like the one used for RHEL, yes, the graph will delay render until all metric facets come through. And in the event of a graph display used for something like RHOSAK each graph will handle its own loading pace